core-web/libs/template-builder/src/lib/components/template-builder/utils/gridstack-utils.ts

```diff
@@ -131,6 +131,49 @@ export function parseFromDotObjectToGridStack(
     })) as DotGridStackWidget[];
 }
 
+export const parseFromGridStackToDotObject = (gridData: DotGridStackWidget[]): DotLayoutBody => {
```

Comment:
We need to test this... in fact the whole file. We missed.

Reply:
There's a test file where we can test this. It just have the parser from DotObject to GridStack, because the rest of the functions are being tested in an implicit way when testing the components and the store. But, if we want to be 100% sure that all of the functions work as expected, we can add those tests, should not take too much effort